index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <nav>
+      <h2><a href="#">Welcome</a></h2>
+      <i class="fa fa-bars" aria-hidden="true"></i>
+    </nav>
```

Comment:
[Optional] As per the project requirements the container of the Toolbar section is supposed to be a _header_ tag, but it seems the previous reviewer missed this reason why it is optional, but I recommend you to make the section to be in a header tag.
![image](https://user-images.githubusercontent.com/62849872/143327165-ee147334-d7d4-4ab9-8604-27dec5b2f790.png)

Reply:
@t-yanick Thank you for noticing my mistake 👍. @t-yanick If you can please attend to this issue as well that would be great and sorry for missing it in the first place.